src/components/sortable/sortable.stories.module.scss

```
@@ -0,0 +1,78 @@
+.chip-item {
+  margin-right: 8px;
+  margin-bottom: 8px;
+}
+
+.chip-item-dragging {
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.chip-drag-layer {
+  :global([class*='drag-preview']) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)